docs/Administrator Guide/Thin-Arbiter-Volumes.md

```diff
@@ -1,5 +1,4 @@
 # Thin Arbiter volumes in gluster
-(Currently supported only by GlusterD2)
```

Comment:
Thanks for the changes. Please update the CLI commands in line num 40 and 44

Reply:
Good catch - I am not sure what they should be changed to, though!

Reply:
The syntax is: ```gluster volume create <volname> replica 2 thin-arbiter 1  <host1>:<data-brick1> <host2>:<data-brick2>....<thin-arbiter-host>:<thin-arbiter-brick-path>```

Example: 
```gluster volume create testvol replica 2 thin-arbiter 1 host1:/bricks/brick1 host2:/bricks/brick2  host3:/bricks/brick3 host4:/bricks/brick4   tahost:/bricks/ta_brick``` will create a 2x2 volume with thin arbiter.

Reply:
@itisravi per your comment at https://github.com/gluster/glusterdocs/commit/50709cd1b6c5a8f09a8ca5bb81bee81a7f43b137#r37170826 - would replacing the set of commands by the link to script suffice?

In that case, the set of changes to the document should be:

- removal of reference to gd2
- adding link to the script

The RN link and sizing of the TA volume we could get done later.

Reply:
Gave my suggestions on the above link. @aspandey @karthik-us Feel free to chime in.